TELCODOCS-2005: Add topics to Telco Day 2 Troubleshooting and Maintenance (Security, Certs, MCO, BMN) (#85961)
openshift/openshift-docs: modules/telco-troubleshooting-security-authentication.adoc

// Module included in the following assemblies:
//
// * edge_computing/day_2_core_cnf_clusters/troubleshooting/telco-troubleshooting-security.adoc
:_mod-docs-content-type: PROCEDURE
[id="telco-troubleshooting-security-authentication_{context}"]
= Authentication

> Determine which identity provider or providers are in your cluster.

Reviewer: Suggest "identity providers" instead of "identity provider or providers" to be less wordy.

For more information about supported identity providers, see "Supported identity providers" in _Authentication and authorization_.

> After you know which provider or providers are configured, you can inspect the `openshift-authentication` namespace to determine where potential issues might be.

Reviewer: ```suggestion
After you know which providers are configured, you can inspect the `openshift-authentication` namespace to determine if there are potential issues.
```
Or consider "determine if potential issues exist" or something similar. I think "might be" is discouraged by IBM Style although I can't find a specific reference.
